helmfile/values/thanos/query.yaml.gotmpl

```
@@ -3,6 +3,7 @@ existingObjstoreSecret: thanos-objectstorage-secret-objstore-secret
 query:
   enabled: true
   replicaCount: {{ .Values.thanos.query.replicaCount }}
+  topologySpreadConstraints: {{- toYaml .Values.prometheus.topologySpreadConstraints | nindent 4 }}
```

```suggestion
  topologySpreadConstraints: {{- toYaml .Values.thanos.query.topologySpreadConstraints | nindent 4 }}
```
shouldn't this be thanos?